js/config.js

```diff
@@ -25,19 +25,19 @@ var lang = 'en';
 //  will show up for each day marked as true.  If the array is or not
 //  provided, all days are assumed to be true.
 //                                                                                                              Sun    Mon    Tues   Wed   Thurs   Fri     Sat   
-var warningTimes = [ { startTime: "7:10", endTime: "7:20", color:"FFA500", message:"Train to Boston",   days:[ false,  true,  true,  true,  true,  true,  false ] },
```

Comment:
I don't think you meant to upload your config.js here.  confg-REMOVE_THIS-.js is intended to be a template, but your config includes your own customizations and API keys, which other people might inadvertently use and wind up overflowing your maximum number of requests.